src/serverMiddleware/routes.js

```diff
@@ -7,10 +7,10 @@
 
 module.exports = [
 	`^/administration/datasources?`,
-	// `^/administration/students/?$`,
-	// `^/administration/students/new/?$`,
-	// `^/administration/teachers/?$`,
-	// `^/administration/teachers/new/?$`,
+	`^/administration/students/?$`,
+	`^/administration/students/new/?$`,
+	`^/administration/teachers/?$`,
+	`^/administration/teachers/new/?$`,
```

Comment:
I think they should still be disabled. @CordlessWool commented them out just two days ago.
```suggestion
	// `^/administration/students/?$`,
	// `^/administration/students/new/?$`,
	// `^/administration/teachers/?$`,
	// `^/administration/teachers/new/?$`,
```

Reply:
pushed solution to branch about that, I don't know why it's still showing in PR